src/main/resources/com/cloudbees/jenkins/support/SupportAction/index.jelly

```diff
@@ -75,7 +75,11 @@
           </f:entry>
         </j:forEach>
         <f:entry>
-          <f:submit value="${%Delete Bundle}"/>
+          <span class="yui-button yui-submit-button submit-button danger">
+            <span class="first-child">
+              <button type="submit">${%Delete Bundle}</button>
```

Comment:
should be "Delete Bundle(s)" as you can select more than one to delete.

Reply:
Technically, you can select logs as well..